graylog2-web-interface/src/stores/grok-patterns/GrokPatternsStore.js

```diff
@@ -47,6 +48,31 @@ const GrokPatternsStore = Reflux.createStore({
       );
   },
 
+  searchPaginated(page, perPage, query) {
+    const url = PaginationURL(ApiRoutes.GrokPatternsController.paginated().url, page, perPage, query);
+    const promise = fetch('GET', qualifyUrl(url))
+      .then((response: any) => {
```

Comment:
Do we really want `any` back from the response or do we know the structure enough to add a type?